packcheck.sh

```diff
@@ -101,7 +101,7 @@ require_cmd () {
 
 # $1: command
 function run_verbose() {
-  echo "$*"
+  echo "$*" 1>&2
```

Comment:
You're redirecting `stdout` to `stderr`, why?

Reply:
This is an informative message. When we redirect the output of `run_verbose` we do not want this to be mixed with the actual output of the command.

Reply:
Ah, You're only redirecting the `echo` of the command. My bad.